using-mobile-extensions/adobe-audience-manager/audience-manager-api-reference.md

```diff
@@ -174,7 +253,7 @@ Audience.getVisitorProfile(visitorProfileCallback);
 {% tab title="iOS" %}
 ### getVisitorProfile
 
-On iOS, the visitor profile is saved in `NSUserDefaults`.
+This API returns the visitor profile that was most recently obtained. Visitor profile is saved in `NSUserDefaults` for easy access across multiple launches of your app. If no signal has been submitted, nil is returned.
```

Comment:
Please reword the 2nd sentence, so that it reads like this:
"For easy access across multiple launches of your app, the visitor profile is saved in SharedPreferences. "